database.mjs

```diff
@@ -0,0 +1,32 @@
+import { createRequire } from 'module'
+const require = createRequire(import.meta.url);
+const { MongoClient } = require('mongodb');
```

Comment:
What is this `createRequire` for?

Reply:
this allows for the require syntax to be used in an mjs file, where it is otherwise undefined.